build.gradle

```diff
@@ -46,7 +46,7 @@ dependencies {
     implementation "io.springfox:springfox-swagger2:${springfoxSwaggerVersion}"
     implementation "io.springfox:springfox-swagger-ui:${springfoxSwaggerVersion}"
     implementation "javax.inject:javax.inject:1"
-    implementation("org.geotools:gt-solr:22.1") {
+    implementation("org.geotools:gt-solr:22.2") {
```

Comment:
@mcalcote Starting a convo to discuss https://github.com/connexta/ion-search/pull/85#issuecomment-558703549:

Idk. I noticed that too. I hoped dependabot recreate would fix it. I doubt it's a real issue. I'd be fine merging this with only a local hero.

Reply:
@ahoffer Moving discussion for https://github.com/connexta/ion-search/pull/85#issuecomment-558708697 here to start a thread:

Does that mean that this PR should be closed?